docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Can you set this to use TF2:</p><p><code></code><code></p><p>try:</p><p>&nbsp;%tensorflow_version 2.x</p><p>except:</p><p>&nbsp;pass</p><p> </p><p>import tensorflow as tf</p><p></code><code></code></p><p> </p><p>We should be able to remove these in the near future</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
@seanpmorgan I thought this snippet fetched tensorflow 2.0

Copied from the tensorflow 2.0 for expert codelab
https://colab.sandbox.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/quickstart/advanced.ipynb